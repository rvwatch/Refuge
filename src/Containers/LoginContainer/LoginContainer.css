.login-wrap {
  width: 300px;
  background-color: rgba(255,255,255,.7);
  display:block;
  margin: 50px auto;
  padding: 40px 0 20px 0;
  text-align: center;
  border-radius: 10px;
  -webkit-box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.35);
  -moz-box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.35);
  box-shadow: 0px 2px 5px 0px rgba(0,0,0,0.35);
}

.login-wrap div {
  margin: 10px 0;
  display: block;
}

.signin-form {
  text-align: left;
  width: 200px;
  margin: auto;
}

.signin-form h2 {
  color: #656565;
  margin: -20px 0 20px 0;
}

.signin-form input {
  background-color: transparent;
  border: none;
  font-size: 18px;
  border-bottom: 1px solid #656565;
}

.signin-form .submit-btn {
  border: 3px solid #656565;
  font-size: 20px;
  background-color: transparent;
  border-radius: 5px;
  padding: 0px 10px 2px 10px;
  margin-top: 20px;
}