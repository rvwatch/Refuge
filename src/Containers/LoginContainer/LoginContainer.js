import React from 'react';
import SignIn from './SignIn/SignIn';
import './LoginContainer.css';

export const LoginContainer = () => {
  return (
    <article className='login-wrap'>
      <SignIn />
    </article>
  );
};
