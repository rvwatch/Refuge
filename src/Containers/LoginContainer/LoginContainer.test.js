import {LoginContainer} from './LoginContainer';

describe('LoginContainer', () => {
  it('should match the snapshot', () => {
    expect(LoginContainer).toMatchSnapshot();
  });
});
