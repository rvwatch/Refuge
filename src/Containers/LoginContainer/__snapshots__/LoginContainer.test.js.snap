// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer should match the snapshot 1`] = `[Function]`;
